### PR TITLE
add: `wsysmon-bin`

### PR DESCRIPTION
**Pull request by 0x262D**

### CODE REVIEW

File: packages/wsysmon-bin/wsysmon-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,30 @@
+maintainer="0x262D <pacstall@copypaste.anonaddy.me>"
+
```
Comment (Henryws) [inline packages/wsysmon-bin/wsysmon-bin.pacscript:2]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]
Does this program have an icon?

**Comment by 0x262D** [comment]
> Does this program have an icon?

This project does not have an icon, but I found a similar project with an icon that could be used. Is it worth it to do so?

Icon
https://github.com/KrispyCamel4u/SysMonTask/blob/master/icons/SysMonTask.png

File: packages/wsysmon-bin/wsysmon-bin.pacscript (hunk 1 above)
Comment (0x262D) [reply packages/wsysmon-bin/wsysmon-bin.pacscript]: Fixed

**Comment by Henryws** [comment]
> > Does this program have an icon?
> 
> This project does not have an icon, but I found a similar project with an icon that could be used. Is it worth it to do so?
> 
> Icon [https://github.com/KrispyCamel4u/SysMonTask/blob/master/icons/SysMonTask.png](https://github.com/KrispyCamel4u/SysMonTask/blob/master/icons/SysMonTask.png?rgh-link-date=2022-08-28T13%3A52%3A51Z)

If you want, sure. However I'd recommend not using other projects icons.

**Review by Henryws — APPROVED** [review]

**Pull request merged**